conformity/rule_settings_schema.go

```diff
@@ -45,6 +45,7 @@ func ExtraSettingSchema() *schema.Schema {
 	return &schema.Schema{
 		Type:     schema.TypeSet,
 		Optional: true,
+		MaxItems: 2,
```

Comment:
❓ hmmm does this break anything? Why limit to 2?

Reply:
Shouldn't be here. Mistook it with Exceptions. I have removed it.